composer.json

```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require":      {
-        "knplabs/gaufrette": "~0.1.7|~0.2|~0.3|~0.4|~0.5",
```

Comment:
Unfortunately, this won't work:
1. the notation is wrong (I assume you wanted to do something like `^0.1`)
2. Composer does not support `^X.Y` notation in the same way for `0.x` branches as for major branches, because of semver:
> 4. Major version zero (0.y.z) is for initial development. Anything may change at any time. The public API should not be considered stable.

Reply:
Thanks, composer also specifies it : 
>>> For pre-1.0 versions it also acts with safety in mind and treats `^0.3` as `>=0.3.0 <0.4.0`.

so I added a `~0.6` ability.